setup.py

```diff
@@ -38,9 +38,9 @@
 # specifying version for `black` separately because it is also used to [check for lint](https://github.com/mlcommons/GaNDLF/blob/master/.github/workflows/black.yml)
 black_version = "23.11.0"
 requirements = [
-    "torch==2.5.0",
+    "torch==2.7.1",
     f"black=={black_version}",
-    "lightning==2.4.0",
+    "lightning==2.5.2",
```

Comment:
I think this needs to be in a separate PR [[ref](https://github.com/mlcommons/GaNDLF/actions/runs/16939634464/job/48006035588?pr=1028)]
```suggestion
    "lightning==2.4.0",
```